client/src/components/InterestModal.vue

```diff
@@ -106,15 +112,25 @@
           });
       },
 
-      trackEvent() {
+      trackEventInterestClick() {
         this.$ga.event({
-          eventCategory: 'Job List',
+          eventCategory: 'Click on InterestModal send button',
+          eventAction: 'click',
+          eventLabel: 'I am really interested',
```

Comment:
Et donc ici tu peux mettre le "on InterestModal send button"
Pas besoin de rappeler le click, c'est déjà dans l'action
Un autre label pourrait être le titre du bouton. Ou "Person interested on this mission"
Ou autres...